### ISSUE TITLE
[Game Crash] Divinity Original sin 2

### ISSUE THREAD

**Issue by Sethox** [lutris/lutris#806]
TL;DR - Not much to go on but I found out that dotNET missing is the cause of the crash. This pastebin gives us nothing. https://pastebin.com/sEJ9Au6a

So a whole day I tried to get the game D:OS2 to work through Winesteam but could not. I tested with older wine versions reinstalled the app (even removing all cache, config files from Lutris to accurately have a fresh install) nothing worked. But I did find something interesting. If I winetricks the dotNet version required to play the game (not the installation steam provides but what winetricks provides) the game actually starts normally, it gets the survey up (just as it would play the first time) but only if I have dotNet installed. Knowing that installing dotNet can break wine I refreshed wine and tried to forcemono. With no such luck (even went out of my way to install the Mono SDK, not wine-mono). Usually the query if I want to install Gecko and Wine-Mono usually pops up when starting up wine for the first time but it did not (checking winetricks wine uninstaller if they were installed, only gecko 32-bit installed).

This is not much to go on but I tried to bisect the issue (only the one instance to the crash). But as that pastebin link shows, not much is given. But my suspicion is still the dotNET libraries are needed which causes the game to crash (NO EVIDENCE OF THIS).

**Comment by strycore** [comment]
you can enable WINEDEBUG output in the game options. Also, is the game 32 bit?

**Comment by Sethox** [comment]
WINEDEBUG, yeah that's right. In all my fustration to get to work I forgot that variable. The game is 64-bit, I tried to install mono 64-bit (http://www.mono-project.com/download/stable/#download-win) with winetricks, same effect (crash). Like I said on my previous post.. I got further when I installed the dotNET with winetricks.

But I will get around to continue to see what causes (when I get time) the crash. Also thanks for reminding me !

**Comment by strycore** [comment]
I don't think there's any way to run 64bit .NET applications at the moment but I would like to be proven wrong on that.

**Comment by cRaZy-bisCuiT** [comment]
Why don't you guys just use the native Linux version? It runs pretty well!

**Comment by Adam-Cass** [comment]
@cRaZy-bisCuiT I think you're confusing Divinity Original Sin 2 with a different Divinity game. I'll link to the steam page of it below.
 [http://store.steampowered.com/app/435150/Divinity_Original_Sin_2/](http://store.steampowered.com/app/435150/Divinity_Original_Sin_2/)

**Comment by strycore** [comment]
Closing as this is not related to the client itself and hasn't received any activity lately.
